main.go

```diff
@@ -77,6 +80,390 @@ func main() {
 					return nil
 				},
 			},
+			{
+				Name:  "create-customer",
+				Usage: "Creates a new customer to go in the customers database, must specify email and state(2 letter code)",
+				Action: func(cCtx *cli.Context) error {
+					if len(os.Args) != 4 {
+						return errors.New("Must specify email and state")
+					}
+					email := os.Args[2]
+					state := os.Args[3]
+					insertStatement := "INSERT INTO Customers (email, state) VALUES (?, ?)"
+					_, err := db.Exec(insertStatement, email, state)
+					if err != nil {
+						log.Fatal(err)
```

Comment:
Since log.Fatal calls `panic` under the hood, we typically don't want to use it since it halts the execution of the program. We can just return errors instead:
```suggestion
						return err
```

Reply:
This applies to more than just this location, but will just comment once here.